contracts/optimistic-ethereum/OVM/execution/OVM_StateManager.sol

```diff
@@ -578,11 +606,69 @@ contract OVM_StateManager is iOVM_StateManager {
         return totalUncommittedContractStorage;
     }
 
+    function wasContractStorageChanged(
+        address _contract,
+        bytes32 _key
+    )
+        override
+        public
+        view
+        returns (
+            bool
+        )
+    {
+        bytes32 item = _getContractStorageItemHash(_contract, _key);
+        return itemStates[item] == ItemState.ITEM_CHANGED;
+    }
+
+    function wasContractStorageCommitted(
+        address _contract,
+        bytes32 _key
+    )
+        override
+        public
+        view
+        returns (
+            bool
+        )
+    {
+        bytes32 item = _getContractStorageItemHash(_contract, _key);
+        return itemStates[item] == ItemState.ITEM_COMMITTED;
+    }
+
 
     /**********************
      * Internal Functions *
      **********************/
 
+    function _getAccountItemHash(
```

Comment:
nit: these could both be a `_getItemHash` with argument-dependent lookups, up to you.

Reply:
Good point, changing.